extensions/replication/ReplicationConfigValidator.js

```diff
@@ -71,7 +71,10 @@ const joiSchema = joi.object({
         concurrency: joi.number().greater(0).default(10),
         mpuPartsConcurrency: joi.number().greater(0).default(10),
         minMPUSizeMB: joi.number().greater(0).default(20),
-        probeServer: probeServerJoi.default(),
+        probeServer: joi.alternatives().try(
+            probeServerJoi,
+            probeServerPerSite
+        ).default({ bindAddress: 'localhost', port: '' }),
```

Comment:
do we need to separate the 2 definitions (probeServerJoi and probeServerPerSite) ?

```suggestion
probeServerJoi = joi.alternatives().try(
            <probeServerJoi>,
            probeServerPerSite
        ).default({ bindAddress: 'localhost', port: '' })
```

the default looks strange, should have a port (which should be an integer?)

Reply:
We need both because we want to have probeServerPersite available only for replication, so I can't set the alternative to probeServerJoi directly..